ntcore/src/main/java/edu/wpi/first/networktables/NetworkTable.java

```diff
@@ -375,28 +445,6 @@ public String getPath() {
     return m_path;
   }
 
-  /**
-   * Save table values to a file. The file format used is identical to that used for SavePersistent.
-   *
-   * @param filename filename
-   * @throws PersistentException if error saving file
-   */
-  public void saveEntries(String filename) throws PersistentException {
```

Comment:
Reduced functionality? Same for `loadEntries`

Reply:
It could be added, but it's not terribly useful, or would need to have a lot of caveats.  In NT3, all nodes had all values.  That's not necessarily the case in NT4--you have to be subscribed to get values (and even see the topics come through), and loadEntries would need to make them all retained (or it would have to publish it).